onnxruntime/core/providers/xnnpack/nn/softmax.h

```diff
@@ -3,15 +3,15 @@
 
 #pragma once
```

Comment:
We do not need headers for these files?

// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License."

Reply:
Seems all files in ORT have similiar declaration headers, Do you mean we don't need these headers in `.h` file?